docs/architecture/baking.md

```diff
@@ -31,18 +31,29 @@ To become a baker, you must create an account to act as a _delegate_, which is a
 Delegates temporarily stake tez as a security deposit to ensure that they are acting according to the rules of the protocol.
 This stake stays locked for a certain number of cycles before the delegate can begin baking.
 Similarly, when bakers stop baking, their stake is unlocked after a certain number of cycles.
+While it is locked, the security deposit may be slashed by the protocol if the baker misbehaves (e.g., proposes or attests two different blocks for the same level).
 
-The delegate must stake at least least 6,000 tez, either from its own account or from tez that delegators delegate to it.
-The more tez a delegate has, both from its own account and from delegators, the more likely it is to be selected to bake a block and thus receive the rewards.
+The delegate must stake at least least 600 tez from its own account,
+and must also represent at least 6,000 tez of "active stake", which is computed
+from the tez in its own account and from the tez of all its delegators accounts.
+The more active stake a delegate represents, the more likely it is to be selected to bake a block and thus receive the rewards.
 
 Bakers must run at least one Tezos node and a baker daemon to go with it.
-These daemons must run at all times with a stable internet connection, because inactive bakers are automatically removed from the network.
+These daemons must run at all times with a stable power source and internet connection, because inactive bakers are automatically removed from the network.
 
 ## Delegating to a baker
 
-If you don't have enough tez to become a baker or don't want to run a baking node, you can delegate tez to a baker, which makes you a _delegator_.
-The delegate doesn't have control over your tez and you can spend your tez at any time or withdraw your delegation, but the tez that you delegate counts toward the amount of tez that the delegate has for baking purposes.
-Delegators receive a share of the delegate's rewards in proportion to the amount of tez that they delegate.
+If you don't have enough tez to become a baker or don't want to run a baking node, you can choose a baker as your delegate, which makes you a _delegator_.
+The delegate doesn't have control over your tez and you can spend your tez at any time or withdraw your delegation, but half of the tez that you delegate counts toward the amount of active stake that the delegate has for baking purposes.
+Delegators receive a share of the delegate's rewards in proportion to the amount of tez that they currently have.
+
```

Comment:
```suggestion

Bakers accept delegated tez because delegated tez counts toward their own tez for certain purposes:

- Delegated tez carries voting power; a delegate's voting power is the sum of its own tez plus the tez delegated to it
- Delegated tez carries baking power; 50% of delegated tez counts toward a delegate's baking power

Delegating carries no risk to the delegator; the delegate has no control over the tez and the delegator is not punished for anything that the delegate does.
```

Reply:
I applied this suggestion manually, because some parts were redundant with the previous paragraph.

Reply:
Got it, removing this suggestion.